scripts/tasks/src/lint-imports.ts

```diff
@@ -60,7 +61,10 @@ export function lintImports() {
   const currentPackageJson = readConfig('package.json');
   const currentMonorepoPackage = currentPackageJson.name;
 
-  return lintSource();
+  return lintSource().catch(err => {
```

Comment:
this was causing some extent of issues especially with node 16 - which throws on uncaught promise rejections. this should improve task execution stability 🫡